bin/setup

```diff
@@ -24,14 +24,14 @@ Dir.chdir APP_ROOT do
   ]
 
   puts "== Copying application.yml =="
-  run "cp config/application.yml.example config/application.yml"
+  run "test -e config/application.yml || cp config/application.yml.example config/application.yml"
```

Comment:
I think we discussed this before. `application.yml` must be allowed to be overwritten because we often make changes to the example file that must be copied over to the local file in order for the app to run locally.